docs/source/docker/integration/docker_with_datasource.rst

```diff
@@ -154,4 +154,54 @@ has the following advantages:
 
 If you want to search all data in your S3 bucket for new samples
 instead of only newly added data,
-then set `datasource.process_all=True` in your docker run command.
\ No newline at end of file
+then set `datasource.process_all=True` in your docker run command.
+
+
+Network traffic
+---------------
+
+Please ensure that the region of your bucket and where you intend to be running the
+Lightly Docker instance are be the same (e.g. `eu-central-1`). If the region is not
+the same there can be `degraded transfer speeds and additional costs will be incurred by AWS <https://aws.amazon.com/premiumsupport/knowledge-center/s3-transfer-data-bucket-instance/>`_!
+This also applies to other cloud providers. We highly recommend using the same region.
+
+
+The worker causes significant network traffic at the following steps:
+
+For image datasets:
+
+- The corruptness check downloads the complete dataset.
+- Training the embedding model downloads the complete dataset once each epoch.
+- Embedding and pretagging each download the non-corrupt dataset.
+- Dumping the selected dataset downloads it.
+- Updating the selected dataset in the Lightly platform
+  will first download all newly selected images to compute their metadata.
+  If you configured this step to upload also thumbnails,
+  it will upload one thumbnail for each newly selected image.
+- Creating the report with images overlayed in the scatterplot will download about 50 images.
+- Uploading the report needs a few MB.
+
```

Comment:
What's missing here is some sort of upper boundary. E.g. if you use all the features of Lightly you will create download traffic from your cloud storage equivalent to (training epochs + 5) x bucket size in GBytes.

And give an example: If you have a dataset consisting of 500'000 images of size 10 GBytes and run Lightly with training an SSL model for 10 epochs you will face at most (10 + 5) x 10 GBytes = 150 GBytes of download traffic.